static/jslicense.html

```diff
@@ -0,0 +1,19 @@
+<html>
+<head>
+	<title>JavaScript license Information</title>
+	<style>a {
+		text-align:center;
+		width:300;
+	}
+	</style>
+<body>
+<h1>JavaScript Web labels table</h1>
+<table id="jslicense-labels1" style="text-align:center; width:500">
+    <tr>
+        <td><a href="/plinth/modules/xmpp/static/jsxc-plinth.js">jsxc-plinth.js</a></td>
```

Comment:
This link doesn't work for me.
The one that works is: /plinth/static/xmpp/jsxc-plinth.js